app/components/search/Sidebar/Sidebar.tsx

```diff
@@ -28,16 +30,42 @@ const Sidebar = ({
     setSearchRadius(evt.target.value);
   };
 
+  const eligibilityNames = useMemo(() => {
+    // This var holds a set of eligibility names pertaining to a given whitelabeled category. This
+    // set can be used to filter the collection of eligibilities returned by Algolia in cases where
+    // we only want to display a specific list of eligibilities for said whitelabeled category.
+
+    // Currently, this is only set up to work with UCSF, but can be expanded upon in the future.
+    const resourceEligibilityGroups = ucsfEligibilityMap?.[categorySlug] ?? [];
+    const flatEligibilities = resourceEligibilityGroups.flatMap(group => (group.eligibilities));
+    return flatEligibilities.map(eligibility => eligibility.name);
+  }, []);
+
   // Currently, the Search Results Page uses generic categories/eligibilities while the
   // Service Results Page uses COVID-specific categories. This logic determines which
   // of these to use as based on the isSearchResultsPage value
   if (isSearchResultsPage) {
     categoryRefinementJsx = <FacetRefinementList attribute="categories" limit={100} mapping={categoriesMapping} />;
     eligibilityRefinementJsx = <FacetRefinementList attribute="eligibilities" limit={100} mapping={eligibilitiesMapping} />;
   } else {
+    const transformEligibilities = (items: any) => {
+      let itemsList = items;
+      if (eligibilityNames.length > 0) {
+        itemsList = items.filter(({ label }: { label: string }) => (
```

Comment:
```suggestion
    const transformEligibilities = (items: { label: string }[]) => {
      let itemsList = items;
      if (eligibilityNames.length > 0) {
        itemsList = items.filter(({ label }) => (
```

Rather than annotating that inner `filter()` arrow function's argument, I think it'd be better to annotate the argument to `transformEligibilities`. Just as a reminder, TypeScript's type system is unlike most common programming languages in that it's [structurally typed](https://en.wikipedia.org/wiki/Structural_type_system), and the object type syntax just means "it must at least have these properties with these types, but could have other properties that we don't know about".